CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+# v1.13.0 (2022-03-??)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to release ([#2829])
+* Add `NoOpMigration` for migrations and migration-helpers
```

Comment:
This can probably be dropped since we added it to v1.12.0 before the release happened. So as far as consumers of Bottlerocket releases go, this was added in the last release and no need to report it again in this release.